db/migrate/20250930111323_create_payment_methods.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class CreatePaymentMethods < ActiveRecord::Migration[8.0]
+  def change
+    create_table :payment_methods, id: :uuid do |t|
+      t.references :organization, type: :uuid, null: false, foreign_key: true
+      t.references :customer, type: :uuid, null: false, foreign_key: true
+      t.references :payment_provider_customer, type: :uuid, null: false, foreign_key: true
+
+      t.string :provider_method_id
+      t.string :method_type, index: true
```

Comment:
@lovrocolic 
When looking at it now, I'm thinking if not to unify the naming here:

`provider_method_id, provider_method_type`
or
`method_id, method_type`